rustls/src/lib.rs

```diff
@@ -306,10 +306,15 @@ use log;
 
 #[cfg(not(feature = "logging"))]
 #[macro_use]
-mod log {
+#[doc(hidden)]
+pub mod log {
+    #[macro_export]
     macro_rules! trace    ( ($($tt:tt)*) => {{}} );
+    #[macro_export]
     macro_rules! debug    ( ($($tt:tt)*) => {{}} );
+    #[macro_export]
     macro_rules! warn     ( ($($tt:tt)*) => {{}} );
+    #[macro_export]
     macro_rules! error    ( ($($tt:tt)*) => {{}} );
 }
```

Comment:
As an alternative to exposing these macros from the rustls crate we could also duplicate the `debug` and `error` macro rules in the `tlsserver-mio.rs` example.

I went with this approach to start but don't feel strongly about it.

Reply:
I changed my mind. It's silly to export this stuff for one example. Force pushed a changed.